src/EnergyPlus/WaterThermalTanks.hh

```diff
@@ -204,50 +204,6 @@ namespace WaterThermalTanks {
         Num
     };
 
-    enum class Fuel
```

Comment:
@dareumnam Have lost the distinction between resource types and fuel types (which are a subset of resource types)? We've had this same issue in other places of the code where one enum class is a subset of a larger group, but we've kept the seemingly duplicate enums.

Reply:
I believe the idea is to restore that distinction, but to do it only once globally. And we need to discuss how to do this because there are various `Fuel` subsets. It was easier to break `Pollutant` out, because that list is used in a much more limited way and is not ambiguous. (Except for `Source` which I am not sure is a pollutant, it seems to me like `Source` should be a variable of type `Fuel` rather than its own type. And then there would also be a `TransmissionLoss` variable. This stuff is wacky).

Reply:
The term "ResourceType" and the full master list was for anything that can be metered. (mtd/mtr ouputs) It was perhaps a misnomer when things like pollutants were added to the list of things you can meter in EnergyPlus. It combined a bunch of overlapping sublists, fuel types, energy flows, electricity meter flows, pollutants, water stuff, etc. And it gave the outputprocessor a single "list" to work with for valid resource types when creating meters. I need to look more closely at the big picture of this PR.

Reply:
I know what you are saying and I agree that we need to reconsider this approach. However, I don't think we should hold up this PR to do so. In general, for things like this, it's good to have smaller "cleanup" PRs, especially if they result in as many textual changes as this one does (you will see this with an upcoming Material cleanup PR from @yujiex). This PR is conceptually small (except for the breakout of Pollutant, but that appears to be non-controversial yes?), but physically quite large. Reconsidering `ResourceType` will be a bigger conceptual PR and we don't want to review the cross-product of that change with this one.